go/vt/mysqlctl/mysqld.go

```diff
@@ -1252,20 +1260,29 @@ func (mysqld *Mysqld) ApplyBinlogFile(ctx context.Context, binlogFile string, re
 		mysqlCmd.Dir = dir
 		mysqlCmd.Env = env
 		mysqlCmd.Stdin = pipe // piped from mysqlbinlog
+
+		mysqlCmd.Stderr = mysqlErrFile
+		log.Infof("ApplyBinlogFile: running mysql command: %#v with errfile=%v", mysqlCmd, mysqlErrFile.Name())
```

Comment:
Just confirming that we want go syntax `%#v` here vs printing out the struct with field names `%+v`. If so, then all good.

Reply:
Yeah, supposed to be `%#v` and this is just copied+pasted from similar calls elsewhere in this file. I'm just maintaining consistency.